GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -231,15 +231,6 @@ T   Translate
 
         public override RevisionGridControl RevisionGridControl { get => RevisionGrid; }
 
-        [Obsolete("For VS designer and translation test only. Do not remove.")]
-#pragma warning disable CS8618 // Non-nullable field must contain a non-null value when exiting constructor. Consider declaring as nullable.
-        private FormBrowse()
-#pragma warning restore CS8618 // Non-nullable field must contain a non-null value when exiting constructor. Consider declaring as nullable.
-        {
-            InitializeComponent();
-            InitializeComplete();
```

Comment:
`InitializeComplete();` is not part of the generated code.
The called functions do nothing if `IsDesignMode` is true - so OK.

Reply:
The default constructor is only needed for the Windows Forms designer, which only needs `InitializeComponent()`.